selene_sdk/targets/genomic_features.py

```diff
@@ -317,6 +319,9 @@ def _query_tabix(self, chrom, start, end):
             The 0-based start position of the query coordinates.
         end : int
             One past the last position of the query coordinates.
+        strand : {'+', '-', '.'}
+            The strand the sequence is located on. '.' is treated as '.'.
```

Comment:
sorry I do have one last thought on this - I think `'.' is treated as '.'` is still too ambiguous for someone who might not look at the code implementation explicitly (especially in other functions where you'd have to trace back to this specific function to see how it gets handled). 

You're basically saying that if the query is `chr, start, end, '.'` we wouldn't retrieve sequences that are `+` or `-`, only `.` right? I think the documentation should say that explicitly. 

That said.. I wonder if this should be handled differently. That is:

- if '+' or '-' is passed in, only retrieve rows with the correct matching strand -- exactly what you are doing now
- if '.' is passed in, treat it the same as strand = None (i.e. change your check to `if strand == '+' or strand == '-'` in line 338) 

does that sound reasonable or is there a situation I'm missing where it would be important to only retrieve rows with `.`?

Reply:
Fix pushed - I agree the handling of '.' was confusing before and think this approach is better! '.' is now treated as None.